christfellowshipchurch/apollos-app
# [Chat] Query and use channelType from API

# Related PRs
- **🛰️ api**: [[Chat] Add channelType to StreamChatType schema #158](https://github.com/christfellowshipchurch/apollos-api/pull/158)
- **🖥️ web**: [[Chat] Query and use channelType from API #193
](https://github.com/christfellowshipchurch/web-app/pull/193)

# Changes or Updates
- Updates queries for `LiveStream` and `GroupItem` to get `streamChatChannel.channelType` from the API
- Use that value ☝️ to initialize event and/or group chat contexts

# Test Instructions
- Make sure **🛰️ api** > [[Chat] Add channelType to StreamChatType schema #158](https://github.com/christfellowshipchurch/apollos-api/pull/158) is merged OR run that branch locally (`channel-type-schema`)
- Open a group detail page
- Tap the **Message Group** button
- 🔸 Verify the chat loads and should show (and/or save new) message history unique to that group

# Screenshots
<img src="https://user-images.githubusercontent.com/1812955/98033952-41e1d400-1de4-11eb-9e2c-067152b11257.jpg" width="560" />
